_assets/javascripts/components/resi-player.js

```diff
@@ -0,0 +1,59 @@
+const isDST = (d) => {
```

Comment:
I see we are creating custom JavaScript time utility methods. I don't have an issue with this, but was wondering if it makes sense at all to use something like the JS library Moment and Moment Timezone. That way we can offload the responsibility for getting time right on a more robust library.